src/Microsoft.ServiceFabric.Actors/Runtime/ActorBase.cs

```diff
@@ -118,7 +118,7 @@ internal IActorManager Manager
         }
 
         /// <summary>
-        /// Override this method to initialize the members, initialize state or register timers. This method is called right after the actor is activated
+        /// Overrides this method to initialize the members, initialize state or register timers. This method is called right after the actor is activated
```

Comment:
@v-rboado  Should be "Override".  A developer would need to override this method.